spec/features/questions/create_spec.rb

```diff
@@ -0,0 +1,33 @@
+require "rails_helper"
+
+RSpec.describe "Creating questions" do
+  it "creates a new question" do
+    visit "/"
+    click_link "New Question"
+    fill_in "Title", with: "Is Capybara Cool?"
+    click_button "Create Rating question"
+
+    within(".flash-notice") do
+      expect(page).to have_content("Your question has been created.")
+    end
```

Comment:
After any blocks in code, it's common to put two line breaks. So space out your `within` blocks within this test.

Reply:
ok